src/sentry/templates/sentry/js-sdk-loader.ts

```diff
@@ -71,7 +70,39 @@ declare const __LOADER__IS_LAZY__: any;
     });
   }
 
-  function injectSdk(callbacks) {
+  function onSentryScriptLoaded() {
+    try {
+      _window.removeEventListener(_errorEvent, onError);
+      _window.removeEventListener(_unhandledrejectionEvent, onUnhandledRejection);
```

Comment:
Problem: Inbetween removing these and waiting for the next tick there might be an error which is not recorded. We need to remove these listeners right before we attach the actual non-buffered once, so in theory right before the call to the actual `Sentry.init`.

As discussed: Let's move it into the monkeypatch init.

Reply:
moved this!